java/BUILD.bazel

```diff
@@ -155,6 +155,14 @@ genrule(
     """,
 )
 
+genrule(
```

Comment:
This is not needed. The `gen_maven_deps` rule will copy the binary files to `java/runtime/native_dependencies`. 
I think the problem this that rule doesn't keep the path prefix. We can fix that rule.

Reply:
This is not for bazel rather than maven, or else the path will be `external/com_github_antirez_redis/redis-server` when running bazel-created jar

Reply:
Sorry I'm confused, so is this rule needed or not needed?

Reply:
Discussed with @chaokunyang offline. We'll keep this rule and use it to remove the prefix of the path. This is better because all bianry files are organized in the `java/runtime/native_dependencies` folder.